openfl/component/aggregator/aggregator.py

```diff
@@ -137,6 +139,8 @@ def __init__(
         self.quit_job_sent_to = []
 
         self.tensor_db = TensorDB()
+        # E.L db path from configuration if exists
+        self.persistent_db = PersistentTensorDB()
```

Comment:
It would be best to make this persistent DB configurable through the plan with a parameter like `experiment_checkpoint`. It can be set to true by default, but there should be a way to disable experiment checkpointing because it increases disk usage (can be in the GB's).